docs/components/power-sensor/_index.md

```diff
@@ -26,22 +27,7 @@ Integrate this component to monitor your power levels.
 
 ## Supported models
 
-To use your power sensor with Viam, check whether one of the following [built-in models](#built-in-models) or [modular resources](#modular-resources) supports your power sensor.
-
-### Built-in models
-
-For configuration information, click on the model name:
-
-| Model                 | Description <a name="model-table"></a>         |
-| --------------------- | ---------------------------------------------- |
-| [`fake`](./fake/)     | a digital power sensor for testing             |
-| [`ina219`](./ina219/) | INA219 power sensor; current and power monitor |
-| [`ina226`](./ina226/) | INA226 power sensor; current and power monitor |
-| [`renogy`](./renogy/) | solar charge controller                        |
-
-### Modular resources
-
-{{<modular-resources api="rdk:component:power_sensor" type="power_sensor">}}
+{{<resources api="rdk:component:power_sensor" type="power_sensor">}}
```

Comment:
Only one model shows up in the rendering for power sensor.

Reply:
fixed